projects/storefrontlib/src/translations/en/common.en.ts

```diff
@@ -25,4 +25,26 @@ export const common = {
     date: 'Date',
     orderNumber: 'Order Number',
   },
+  messages: {
```

Comment:
It would be good no to mix features into one `messages` key.
- I think `httpHandlers` should be a grouping key in `common` chunk for http interceptors 
- other keys should placed in the existing namespaces of features that use those keys (i.e. forgotPassword, pwa, addressForm, paymentForm, etc.)